modules/ting/src/TingObjectInterface.php

```diff
@@ -231,4 +251,25 @@ public function getVersion();
    *   Relations.
    */
   public function isPartOf();
+
+  /**
+   * Gets the material type.
+   *
+   * Eg. "Bog"
+   * Eg. "Lydbog (cd)"
```

Comment:
:+1: Appreciate the examples. It helps to understand that this is human readable values and not some kind of type identifiers.